eq-author/src/App/ErrorBoundary/index.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import PropTypes from "prop-types";
+import NotFound from "App/NotFoundPage";
+import { ERR_PAGE_NOT_FOUND } from "constants/error-codes";
+
+export default class ErrorBoundary extends React.Component {
+  static propTypes = {
+    children: PropTypes.object.isRequired, // eslint-disable-line
```

Comment:
Can we use `Proptypes.node` here, save having to disable the line?

Reply:
Done